src/debugger_requests.jl

```diff
@@ -128,6 +128,15 @@ function set_compiled_functions_modules!(items::Vector{String})
 
     # user wants these compiled:
     for acc in items
+        if acc == "ALL_MODULES_EXCEPT_MAIN"
+            for mod in values(Base.loaded_modules)
```

Comment:
I noticed in some of my scripts that the _number of modules increases while running_. That means these additional modules will not be set to compiled.